global_translations.R

```diff
@@ -38,3 +38,27 @@ complete_trans_df <- subset(final_df, untranslated_count == 0, select = -c(fuzzy
 
 # Untranslated Packages with respective languages associated
 complete_untrans_df <- subset(final_df, translated_count == 0, select = -c(fuzzy_count))
+
+# R Language Translation Team Contact details
+library(rvest)
+library(stringr)
+
+translation_teams <- read_html("https://developer.r-project.org/TranslationTeams.html") |>
```

Comment:
Consider trying to future-proof this a bit by adding some assertions about the existence/structure of the page.

What happens if the page moves? Or the simple one-table structure changes?

Feel free to mark this for follow-up

Reply:
Sure Will do that. Thanks for reminding that.

Reply:
For now, it may help to state your assumptions about page structure as a comment, e.g.

```r
# The TranslationTeams page has a single HTML table with
#   with two columns, Language and Contact. Contact is
#   formatted like $MEMBERS <$CONTACT>
#   (or "No current maintainer" if there is no maintainer).
```

Reply:
I can add for now. If this is good to go
`read_html("https://developer.r-project.org/TranslationTeams.html") |>
  html_node("table , th") |>
  html_table()
`

Reply:
`,th` adds as a parameter for the class which the table contains